### PR TITLE
Fix #220: Add test for Pageable for activationList

### LINKED ISSUE #220: Add test for Pageable for activationList

### PR DESCRIPTION
**Pull request by jandusil**
- Add testGetActivationListForUserPagination to test pageable behavior

### CODE REVIEW

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java [hunk 1]
```diff
@@ -245,6 +243,40 @@ void activationListForUserTest() throws PowerAuthClientException {
...
+        request.setUserId(config.getUserV31());
+        request.setApplicationId(config.getApplicationId());
+
+        // Create multiple activations for the test user
+        for (int i = 0; i < 10; i++) {
+            powerAuthClient.initActivation(request.getUserId(), request.getApplicationId());
+        }
+
+        // Prepare the query parameters for the first page
+        Map<String, String> queryParams1 = new HashMap<>();
```
Comment (banterCZ) [inline powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java:259]: I prefer using the modifier `final` (everywhere) to ensure we are not reassigning the variable. Moreover, here you may use the new API `Map#of()`.

**Comment by jandusil** [comment]
Waiting for [https://github.com/wultra/powerauth-server/issues/752](url)

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java [hunk 2]
```diff
@@ -248,6 +249,55 @@ void activationListForUserTest() throws PowerAuthClientException {
         assertNotEquals(0, listResponse.size());
     }
 
+    @Test
+    void testGetActivationListForUserPagination() throws PowerAuthClientException {
+        // Prepare the base GetActivationListForUserRequest
+        GetActivationListForUserRequest baseRequest = new GetActivationListForUserRequest();
```
Comment (banterCZ) [inline powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java:255]: Let's use the modifier `final` everywhere to ensure we are not reassigning the variable.

**Review by banterCZ — APPROVED** [review]
Looks good.

**Pull request merged**